assets/scss/components/_breadcrumbs.scss

```diff
@@ -1,15 +1,35 @@
-
 /* ==========================================================================
    #BREADCRUMBS
    ========================================================================== */
 
 .c-breadcrumbs {
-    @include container();
+    padding: 0;
 }
 
-.c-breadcrumbs ol {
-    @include list-bare();
-
+.c-breadcrumbs__list {
     display: flex;
     gap: rem-calc(10px);
+    margin: 0;
+    list-style-type: none;
+}
+
+.c-breadcrumbs__item {
+    margin: 0;
+}
+
+.c-breadcrumbs__link {
+    color: $color-primary-300;
+    font-family: $font-semi-bold;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+    text-decoration-line: underline;
+}
+
+.c-breadcrumbs__active {
```

Comment:
@mgbelegu this is a great candidate for a modifier instead of an element. I suggest `c-breadrcumbs--active` or `c-breadcrumbs__link--active`